gcsfs/core.py

```diff
@@ -905,6 +905,31 @@ def cat(self, path):
                     assert md5(r.content).digest() == md, "Checksum failure"
         return r.content
 
+    @_tracemethod
+    def get_dir(self, rpath, lpath, blocksize=5 * 2 ** 20):
```

Comment:
Why this name? I would have thought perhaps `sync` or `get(path, recursive=True)`.